.eslintrc.json

```diff
@@ -1,37 +1,48 @@
 {
-    "extends": "google",
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-      "ecmaVersion": "2017",
-      "sourceType": "module"
-    },
-    "plugins": [
-      "@typescript-eslint"
-    ],
-    "rules": {
-      "max-len": ["error", {
+  "extends": ["plugin:prettier/recommended", "google"],
```

Comment:
So the problem with doing that is you end up losing some linting warnings. For example,

> shouldn't this be true? we should use template literals for any interpolation (maybe we don't yet have any?)

This should be `false` because it is enforcing that we don't unnecessarily use backticks. It also, warns if you use `"` if you have it set to single quotes. See https://github.com/prettier/eslint-config-prettier#forbid-unnecessary-backticks

Reply:
my point is we would instead just turn the stylistic rules off... eslint doesn't enable many by default (a couple) and the ones google enables don't usually conflict with prettier.

i.e. we shouldn't be linting quotes when prettier knows how to rewrite them to the correct style already

Reply:
prettier isn't going to fix unnecessary escapes though. So now we have no warning of such a thing by removing this.

Reply:
edited above comment as if we follow what we did in the wc plugin, turning off `indent` and `quote-props` does the job. with the current eslintrc from master